### PR TITLE
React Hands On complete

### PR DESCRIPTION
**Pull request by asish918**
## Drift Video
[Drift](https://video.drift.com/v/abX3gnt5fUY3KGCZGRhMIzqs5XHX9EPXuGNgYLHwlIzQ)

## Screenshots

Original UI (Desktop)
Screenshot 2024-02-19 at 22 47 46

Original UI (Mobile View - It is scrollable)
Screenshot 2024-02-19 at 22 49 57

## I tried to enhance the UI with box-shadows and icons

Enhanced UI (Desktop)
Screenshot 2024-02-19 at 23 11 44

Enhanced UI (Mobile View)
Screenshot 2024-02-19 at 23 14 08

### CODE REVIEW

**Comment by asish918** [comment]
| Metric | Value |
| ------ | ----- |
| alert_status | <b>:white_check_mark: OK</b> |
| bugs | 0 |
| code_smells | 1 |
| reliability_rating | A (0 Bugs)<br/>Note: A being best, E being worst |
| security_rating | A (0 Vulnerabilities)<br/>Note: A being best, E being worst |
| vulnerabilities | 0 |

File: frontend/homework/9/src/components/skills/Skills.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { FaList } from "react-icons/fa6";
+import { Skills } from '../../apptypes'
+import './Skills.scss'
+
+interface SkillsProps {
+    skills: Skills[];
+}
```
Comment (Nitesh2503) [inline frontend/homework/9/src/components/skills/Skills.tsx:7]: Could be stored in the different interface file

File: frontend/homework/9/src/components/skills/Skills.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+export default function SkillSection({ skills }: Readonly<SkillsProps>) {
+    return (
+        <section className="skills">
+            <h1 className='skills__heading'>
+                <FaList className="icon" />
+                Skills
+            </h1>
+
```
Comment (Nitesh2503) [inline frontend/homework/9/src/components/skills/Skills.tsx:16]: Format the code properly

**Comment by Nitesh2503** [comment]
- Nice work on the UI
- Proper commit messages must done

**Pull request closed without merging**